src/lib.rs

```diff
@@ -56,6 +56,14 @@ use util::{TypeLoad, TypeSave};
 #[cfg(all(feature = "openssl", feature = "sev"))]
 use certs::sev::sev;
 
+#[cfg(feature = "openssl")]
+use certs::sev::ca::{Certificate, Chain as CertSevCaChain};
```

Comment:
Since CA chains aren't changed between legacy SEV and SEV-SNP, we should extract the `ca` modules of `certs::sev` and `certs::snp` and only conditionally include what's different between the generations.

Reply:
That is a good idea, but a bit outside of the scope of these changes. Do you want me to introduce those changes here, or should we make an issue and a new PR for it?

Reply:
@tylerfanelli and I discussed this, it is an issue with PEM vs legacy cert format.